src/log.c

```diff
@@ -121,6 +121,9 @@ int log_internal(int level, int error, const char *file, int line, const char *f
 {
     va_list ap;
     int r;
+    int _level = (level);
+    if (log_get_max_level() < LOG_PRI(_level))
+        return -error;
```

Comment:
There is no log_full_errno in csd. Maybe we could use same version from mavlink-router, but that is not the case right now.